### ISSUE TITLE
Load a dataset in the powergrid case without grid2op

### ISSUE THREAD

**Issue by daviddanan** [Mleyliabadi/LIPS#125]
I wanted to load a dataset using existing data for the power grid case, here is a snippet

```
from lips import GetRootPath
from lips.dataset.powergridDataSet import PowerGridDataSet
from lips.config.configmanager import ConfigManager

if __name__ == '__main__':
    configFilePath=GetRootPath()+os.path.join("..","tests","data","powergrid","l2rpn_case14_sandbox","Benchmark1","train")
    CONFIG_PATH_BENCHMARK=GetRootPath()+os.path.join("..","configurations","powergrid","benchmarks","l2rpn_case14_sandbox.ini")

    myConfigManager=ConfigManager(CONFIG_PATH_BENCHMARK)
    myDataset=PowerGridDataSet(config=myConfigManager,
                 name="train")
```

However, i got the error when i tried to import the dedicated dataset class
```
ModuleNotFoundError: No module named 'grid2op'
```

Is it really mandatory to have this package for loading existing (and potentially external) data? Same question for Grid2opSimulator.

If it's not, as these module are only used to generate the data, we could for instance remove the generate method from this class and implement the generation in its own dedicated module (it goes without saying but the same should be done for every usecase).
What do you think?

There are other possibilities (move the problematic imports in the generate method, create new module whose only purpose would be to import case specific module if they are required and available) but that would not be my first choice.

**Comment by Mleyliabadi** [comment]
Yes, completely agree to not force the import of use case libraries if they are not required. And, creating a separate module for generating some data seems also relevant.

**Comment by daviddanan** [comment]
Great, i will give it a try for something that would fit our needs for the powergrid and pneumatic cases at least.
It seems doable
